docs/source/tutorial/recurrentnet.rst

```diff
@@ -291,21 +291,40 @@ Backprop Through Time is implemented as follows.
 
 .. code-block:: python
 
-   def update_bptt(updater):
-       loss = 0
-       for i in range(35):
-           batch = train_iter.__next__()
-           x, t = chainer.dataset.concat_examples(batch)
-           loss += model(chainer.Variable(x), chainer.Variable(t))
-
-       model.cleargrads()
-       loss.backward()
-       loss.unchain_backward()  # truncate
-       optimizer.update()
-
-   updater = training.StandardUpdater(train_iter, optimizer, update_bptt)
-
-In this case, we update the parameters on every 35 consecutive words.
+   class BPTTUpdater(training.StandardUpdater):
+
+       def __init__(self, train_iter, optimizer, bprop_len, device):
```

Comment:
Could you remove `device` argument? This is not essential for this tutorial and may confuse users.